ios/MeasureSDK/MeasureSDKTests/SessionManagerTests.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SessionManagerTests.swift
+//  MeasureSDKTests
+//
+//  Created by Adwin Ross on 02/09/24.
+//
+
+import XCTest
+@testable import MeasureSDK
+
+final class SessionManagerTests: XCTestCase {
```

Comment:
The following public methods must also have corresponding unit tests:
* start()
* applicationDidEnterBackground()
* applicationWillTerminate()
* shouldMarkSessionForExport()

Also try covering the case where getting the session ID throws an exception.